store/presenters/pagination.go

```diff
@@ -0,0 +1,46 @@
+package presenters
+
+import "fmt"
+
+// HALResource is a wrapper for any collection of resources that is paginated.
+type HALResource struct {
+	Data  interface{} `json:"data"`
```

Comment:
Do we plan to implement the `HAL` specification ourself or is there a library we can lean on? e.g. https://github.com/nvellon/hal

I do know that the JSONAPI has a lot of library support, including from [google](https://github.com/google/jsonapi)

Reply:
Nah I'd probably look into supporting https://github.com/google/jsonapi